pluggy.py

```diff
@@ -581,7 +595,7 @@ def varnames(func):
         try:
             func = func.__init__
         except AttributeError:
-            return ()
+            return (), ()
```

Comment:
@nicoddemus if we make it private, and introduce a public wrapper with a deprecationwarning and the old behaviour, its fine as well